### PR TITLE
Remove -it for docker exec command

### PR DESCRIPTION
**Pull request by hosfeld**
docker exec -it adds \r\n newlines on each line, making shell parsing more difficult. 

See https://github.com/osism/ansible-collection-services/issues/1265

### CODE REVIEW

**Comment by berendt** [comment]
Remove it everywhere + add a release note with `reno new remove-docker-it-arguments` or something like this.

File: roles/openstackclient/templates/openstack.j2 [hunk 1]
```diff
@@ -2,17 +2,8 @@
 
 # DO NOT EDIT THIS FILE BY HAND -- YOUR CHANGES WILL BE OVERWRITTEN
 
-INTERACTIVE=${INTERACTIVE:-false}
 PARAMETERS=${PARAMETERS:-}
 
-if [[ -t 0 ]]; then
```
Comment (osfrickler) [inline roles/openstackclient/templates/openstack.j2:8]: This tests whether stdin is a terminal. So if you want to have the output in unix format, just invoke `openstackclient </dev/null`

**Review by osfrickler — COMMENTED** [review]
I'm not sure whether this is a good idea. It breaks using the client in interactive mode, which may or may not be very useful, but at least it is an option currently.

File: roles/openstackclient/templates/openstack.j2 (hunk 1 above)
Comment (hosfeld) [reply roles/openstackclient/templates/openstack.j2:8]: We can either leave the current settings and remove the \r\n from output:
`for i in $(openstack server list -c ID -f value | tr -d "\r"); do openstack server show $i; done`

Or we change the default behaviour of the docker-wrapper-scripts to not containing carriage-returns for easier scripting usability (and maybe add an additional flag if you really want to run an interactive session?)

Comment (osfrickler) [reply roles/openstackclient/templates/openstack.j2:8]: Not sure what is wrong with adding the `</dev/null`, but maybe https://github.com/osism/ansible-collection-services/pull/1268 suits you better?

**Pull request closed without merging**